domenetjenester/brevbestiller/src/test/java/no/nav/foreldrepenger/melding/brevmapper/brev/innvilgelsefp/UtbetalingsperiodeMapperTest.java

```diff
@@ -386,18 +391,70 @@ public void skal_ikke_finne_periode_med_ikke_omsorg() {
     }
 
     @Test
-    public void skal_hente_prioritert_utbetalingsgrad_fra_arbeidsforholdet_med_høyest() {
-        //TODO(JEJ) når det evt. blir avklart at vi løser det slik
+    public void skal_hente_prioritert_utbetalingsgrad_fra_arbeidsforholdet_med_gradering_om_finnes() {
+        List <Arbeidsforhold> arbeidsforholdListe =
+                List.of(opprettArbeidsforhold(true, Prosent.of(BigDecimal.valueOf(20)), Prosent.of(BigDecimal.valueOf(80))),
+                        opprettArbeidsforhold(false, Prosent.of(BigDecimal.valueOf(0.0)), Prosent.of(BigDecimal.valueOf(100))),
```

Comment:
Her bør du muligens unngå at det er første arbeidsforhold som er gradert, slik at du også tester at den faktisk finner frem det graderte arbeidsforholdet, og ikke bare tar det første? (altså bare bytte om på rekkefølgen på de to)

Reply:
har oppdatert